modules/oe_content_entity/modules/oe_content_entity_contact/src/Entity/Contact.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_content_entity_contact\Entity;
+
+use Drupal\oe_content_entity\Entity\EntityBase;
+
+/**
+ * Defines the Contact entity.
+ *
+ * @ingroup oe_content_entity_contact
+ *
+ * @ContentEntityType(
+ *   id = "oe_contact",
+ *   label = @Translation("Contact"),
+ *   bundle_label = @Translation("Contact type"),
+ *   handlers = {
+ *     "view_builder" = "Drupal\Core\Entity\EntityViewBuilder",
+ *     "list_builder" = "Drupal\oe_content_entity\EntityListBuilder",
+ *     "views_data" = "Drupal\views\EntityViewsData",
+ *     "translation" = "Drupal\content_translation\ContentTranslationHandler",
+ *     "form" = {
+ *       "default" = "Drupal\oe_content_entity\Form\EntityForm",
+ *       "add" = "Drupal\oe_content_entity\Form\EntityForm",
+ *       "edit" = "Drupal\oe_content_entity\Form\EntityForm",
+ *       "delete" = "Drupal\Core\Entity\ContentEntityDeleteForm",
+ *     },
+ *     "route_provider" = {
+ *       "html" = "Drupal\Core\Entity\Routing\AdminHtmlRouteProvider",
+ *     },
+ *   },
+ *   base_table = "oe_contact",
+ *   data_table = "oe_contact_field_data",
+ *   revision_table = "oe_contact_revision",
+ *   revision_data_table = "oe_contact_field_revision",
+ *   translatable = TRUE,
+ *   admin_permission = "manage custom content entities",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "revision" = "vid",
+ *     "bundle" = "bundle",
+ *     "uuid" = "uuid",
+ *     "label" = "name",
+ *     "uuid" = "uuid",
+ *     "langcode" = "langcode",
+ *     "published" = "status",
+ *     "created" = "created",
+ *     "changed" = "changed",
+ *   },
+ *   revision_metadata_keys = {
+ *     "revision_user" = "revision_uid",
+ *     "revision_created" = "revision_timestamp",
+ *     "revision_log_message" = "revision_log"
+ *   },
+ *   bundle_entity_type = "oe_contact_type",
+ *   field_ui_base_route = "entity.oe_contact_type.edit_form",
+ *   links = {
+ *     "canonical" = "/admin/content/oe_contact/{oe_contact}",
+ *     "add-page" = "/admin/content/oe_contact/add",
+ *     "add-form" = "/admin/content/oe_contact/add/{oe_contact_type}",
+ *     "edit-form" = "/admin/content/oe_contact/{oe_contact}/edit",
+ *     "delete-form" = "/admin/content/oe_contact/{oe_contact}/delete",
+ *     "collection" = "/admin/content/oe_contact",
```

Comment:
Prepend all URLs with `/admin/content/custom-entities/` so that we get the proper breadcrumb when managing entities. do the same on all other definitions too.

Reply:
On a second thought we will go for a much more simple (and Drupal core-compliant) set of paths.